lib/parse/datetime/parsers.ex

```diff
@@ -230,10 +230,10 @@ defmodule Timex.Parse.DateTime.Parsers do
   ISO 8601 date/time format with timezone information.
   With zulu: true, assumes UTC timezone.
   Examples:
-    2007-08-13T16:48:01+0300
+    2007-08-13T16:48:01+03:00
     2007-08-13T13:48:01Z
   """
-  def iso8601(opts \\ []) do
+  def iso8601_extended(opts \\ []) do
```

Comment:
As mentioned above, let's keep the old API side by side with the new to preserve behaviour. We can deprecate it later.